grapher/controls/Controls.tsx

```diff
@@ -237,52 +235,103 @@ export class FilterSmallCountriesToggle extends React.Component<{
-            return { label, value }
-        })
+    render(): JSX.Element {
+        return (
+            <Tippy
+                content={this.content}
+                interactive={true}
+                trigger={"click"}
+                placement={"bottom-start"}
+                arrow={false}
+            >
+                <div
+                    className="FacetStrategyDropdown"
+                    style={{ width: this.dropDownWidth }}
+                >
+                    {facetStrategyLabels[this.facetStrategy]}
+                    <div>
+                        <FontAwesomeIcon icon={faChevronDown} />
+                    </div>
+                </div>
+            </Tippy>
+        )
+    }
+
+    @computed get strategies(): FacetStrategy[] {
+        return (
+            this.props.manager.availableFacetStrategies || [
+                FacetStrategy.none,
+                FacetStrategy.entity,
+                FacetStrategy.column,
+            ]
+        )
     }
 
-    @computed get facet(): FacetStrategy {
-        return this.props.manager.facet || FacetStrategy.none
+    @computed get dropDownWidth(): string {
+        const maxWidth = Math.max(
+            ...this.strategies.map(
+                (s) =>
+                    Bounds.forText(facetStrategyLabels[s], { fontSize: 12 })
+                        .width
+            )
+        )
+        return `${maxWidth + 45}px` // leave room for the chevron
     }
 
-    @action.bound onChange(
-        selected: ValueType<{ label: string; value: string }>
-    ): void {
-        this.props.manager.facet = asArray(selected)[0].value as FacetStrategy
+    // A hovering visual display giving options to be selected from
+    @computed get content(): JSX.Element {
+        const parts = this.strategies.map((value: FacetStrategy) => {
+            // There are only ever two strategies, none + (country OR entity).
+            const label = facetStrategyLabels[value]
+            const selected = value == this.facetStrategy ? " selected" : ""
+            const children =
+                value === FacetStrategy.none ? (
+                    // a single solid block
+                    <div className="FacetStrategyPreview-none-child"></div>
+                ) : (
+                    // a 3x2 grid of squares
+                    <>
+                        <div className="FacetStrategyPreview-split-child"></div>
+                        <div className="FacetStrategyPreview-split-child"></div>
+                        <div className="FacetStrategyPreview-split-child"></div>
+                        <div className="FacetStrategyPreview-split-child"></div>
+                        <div className="FacetStrategyPreview-split-child"></div>
+                        <div className="FacetStrategyPreview-split-child"></div>
+                    </>
+                )
+            return (
+                <div
+                    className={"FacetStrategyOption" + selected}
```

Comment:
There's a `classnames` library we use that allows you to do something like this:

```suggestion
                    className={classnames({
                        "FacetStrategyOption": true,  
                        "selected": value === this.facetStrategy 
                    })}
```

Reply:
Done